src/model/entities/show-style-variant.ts

```diff
@@ -0,0 +1,6 @@
+export interface ShowStyleVariant {
+  _id: string
```

Comment:
We are using `id`, not `_id` for our entities.

Reply:
Fixed in the upcomming commit.

Reply:
Has been updated to `id`, and mapping from Mongo version to our entity added to MongoEntityConverter.